packages/graphql-cli-validate-magento-pwa-queries/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Adobe Inc.
```

Comment:
Is this an official thing?

Reply:
Published to NPM so I was just covering all the bases 🤷‍♂️